org/vulpea-agenda.el

```diff
@@ -56,13 +110,16 @@ tasks."
         (file-name-directory buffer-file-name))))
 
 (defun vulpea-project-files ()
-  "Return a list of note files containing Project tag."
-  (seq-map
-   #'car
-   (org-roam-db-query
-    [:select file
-     :from tags
-     :where (like tags (quote "%\"Project\"%"))])))
+  "Return a list of note files containing 'project' tag." ;
+  (seq-uniq
+   (seq-map
+    #'car
+    (org-roam-db-query
+     [:select [nodes:file]
+      :from tags
+      :left-join nodes
+      :on (= tags:node-id nodes:id)
+      :where (like tag (quote "%\"project\"%"))]))))
 
```

Comment:
We might exploit roam v2's query feature to get rid of the need of this package. Should revisit this.